rn/Teacher/src/canvas-api/httpClient.js

```diff
@@ -126,6 +128,7 @@ function xhr (method: Method, url: string, data: Body, config: ApiConfig = {}) {
             resolve(response)
             break
           case 'error':
+            console.log(request)
```

Comment:
Do you want to leave these console.logs in here?

Reply:
I'd like them in there unless there's a reason not to. Makes tracking down all the "unknown error" alerts easier.